### PR TITLE
RBTA-53-registration-email/password check-trusevich

### PR DESCRIPTION
**Pull request by itrusevich**
Added tests for registration-email/password check. I reused some variables, data, helpers from Sergey @Miksyuk PR which wasn't merged when I created this PR. 

https://jira.pasv.us/projects/RBTA/issues/RBTA-53?filter=allissues&orderby=created+ASC

### CODE REVIEW

File: wdio.conf.js [hunk 1]
```diff
@@ -3,16 +3,17 @@ exports.config = {
...
-    './test/specs/reg/login.js',
-    './test/specs/reg/all-issues.js',
-    './test/specs/reg/global-header.js',
-    './test/specs/reg/global-footer/general.js',
-    './test/specs/reg/global-footer/design.js',
-    './test/specs/reg/BugList/General/BugList-General.js',
-    './test/specs/reg/registration/general.js',
-    './test/specs/reg/login/error.js',
-    './test/specs/reg/forgot-password.js'
+    // './test/specs/reg/client.js',
```
Comment (Miksyuk) [inline wdio.conf.js]: Uncomment tests pls

File: test/specs/reg/registration/email-password-check.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        help.openRegistration();
+        $(sel.firstName).addValue(regData.firstName);
+        $(sel.lastName).addValue(regData.lastName);
+        $(sel.email).addValue(loginData.email);
+        $(sel.confirmEmail).addValue(loginData.email);
+        $(sel.password).addValue(loginData.pass);
+        $(sel.confirmPassword).addValue(loginData.pass);
+        $(sel.registerMe).click();
+        let actual = $(sel.error).getText();
+        assert.equal(actual, exp.errorUserExists);
```
[the remark this answers is not in the document]
Comment (itrusevich) [reply test/specs/reg/registration/email-password-check.js]: @Miksyuk I'm checking registered before user email here and I'm using the test email from login data, not from registration data. So, yes it's working as email is the same in email and in confirmEmail fields.

File: test/specs/reg/registration/email-password-check.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    });
+
+    it('Emails do not match => error message', function () {
+        browser.refresh();
+        help.openRegistration();
+        help.registration();
+        help.fieldClear(sel.confirmEmail);
+        $(sel.email).addValue(regData.email + '1');
+        $(sel.registerMe).click();
```
[the remark this answers is not in the document]
Comment (itrusevich) [reply test/specs/reg/registration/email-password-check.js]: Yes, you're right I don't need +1. It's excessive. I'll correct.

File: test/specs/reg/registration/email-password-check.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+    });
+
+    it('Passwords do not match => error message', function () {
+        browser.refresh();
+        help.openRegistration();
+        help.registration();
+        help.fieldClear(sel.confirmPassword);
+        $(sel.password).addValue(regData.password +'1');
+        $(sel.registerMe).click();
```
[the remark this answers is not in the document]
Comment (itrusevich) [reply test/specs/reg/registration/email-password-check.js]: It doesn't matter here as long as passwords are different but yes, I'll correct so it looks consistent.

File: test/specs/reg/registration/email-password-check.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Miksyuk) [reply test/specs/reg/registration/email-password-check.js]: oh sorry)

File: wdio.conf.js (hunk 1 above)
Comment (itrusevich) [reply wdio.conf.js]: > tests look great. but seems like you forgot to uncomment everyone else's tests after yours were run.

Corrected. Thanks!

**Comment by artsenius** [comment]
Why so many conflicts?

**Comment by itrusevich** [comment]
> Why so many conflicts?

@artsenius I guess, it happens when different people are testing different sections of the same process/sub-suite(registration in my case). We are working in different files but we use the same selectors, expected and etc files.

**Comment by itrusevich** [comment]
@artsenius all current conflicts resolved. Please review.

**Comment by artsenius** [comment]
One more conflict!

**Pull request merged**